app/views/layouts/shared/_header.html.erb

```diff
@@ -1,15 +1,17 @@
+<%= govuk_header %>
 <%=
-  govuk_header(service_name: "Register for a national professional qualification", service_url: root_path) do |header|
+  govuk_service_navigation do |sn|
```

Comment:
minor nit but could this be renamed to `nav` or `navigation` instead of `sn` - forces the future developer to find the 'opening' line to determine what `sn` is